lighthouse-cli/test/smokehouse/frontends/lib.js

```diff
@@ -13,7 +13,7 @@
 
 /* eslint-disable no-console */
 
-import cloneDeep from 'lodash.clonedeep';
+import { cloneDeep } from 'lodash';
```

Comment:
This errors because apparently lodash is only commonjs.

Reply:
I would assume that `lodash.clonedeep` is also only commonjs. What is the error?

Reply:
The former worked because it was a default import, but the latter is a named import. ESM and tools and automatically coerce commonjs default imports to ESM, but not named.

see the logs in the GitHub Actions CI for the exact error message, or run `yarn start https://www.example.com` locally.